sql/src/main/java/io/crate/planner/CreateViewPlan.java

```diff
@@ -53,9 +59,10 @@ public void executeOrFail(DependencyCarrier dependencies,
                               SubQueryResults subQueryResults) {
 
         User owner = createViewStmt.owner();
+        String formattedQuery = SqlFormatter.formatSql(createViewStmt.query(), makeExpressions(params));
```

Comment:
Should we do another unbound analyze on the formattedQuery, to make sure the parameters don't cause some validation errors?